cpp-data-aggregation/aggregation_consumer.cpp

```diff
@@ -71,11 +64,16 @@ void run_aggregation_consumer(std::ofstream& timing_file,
             bool list_is_ready = client.poll_list_length(list_name,
                                                          list_length,
                                                          5, 100000);
-            if(!list_is_ready)
+            if(!list_is_ready) {
+                log_error(context, LLDebug, "There was an error in the "\
+                                         "aggregation scaling test.  "\
+                                         "The list never reached size of " +
+                                         std::to_string(list_length));
                 throw std::runtime_error("There was an error in the "\
                                          "aggregation scaling test.  "\
                                          "The list never reached size of " +
                                          std::to_string(list_length));
+            }
```

Comment:
Suggest putting the message into a string so that you don't have a duplicate it. That will ease future maintenance, letting people make changes in just one place